rpm/libglibutil.spec

```diff
@@ -1,41 +1,50 @@
-Name: libglibutil
-
-Version: 1.0.80
-Release: 0
-Summary: Library of glib utilities
-License: BSD
-URL: https://github.com/sailfishos/libglibutil
-Source: %{name}-%{version}.tar.bz2
+# The main package is now named libglibutil1
+Name:           libglibutil1
+Version:        1.0.80
+Release:        0
+Summary:        Library of glib utilities
+License:        BSD
+URL:            https://github.com/sailfishos/libglibutil
+# The Source tarball name is still libglibutil
+Source:         libglibutil-%{version}.tar.bz2
 
 %define glib_version 2.32
 
-BuildRequires: pkgconfig
-BuildRequires: pkgconfig(glib-2.0) >= %{glib_version}
+BuildRequires:  pkgconfig
+BuildRequires:  pkgconfig(glib-2.0) >= %{glib_version}
 
 # license macro requires rpm >= 4.11
-BuildRequires: pkgconfig(rpm)
+BuildRequires:  pkgconfig(rpm)
 %define license_support %(pkg-config --exists 'rpm >= 4.11'; echo $?)
 
 # make_build macro appeared in rpm 4.12
 %{!?make_build: %define make_build make %{_smp_mflags}}
 
-Requires: glib2 >= %{glib_version}
+# Requirements are now in the main package
```

Comment:
When you look on a file, everything is "now" like it is. Such comments here don't help the reader much.